resources/developer-mode.mjs

```diff
@@ -168,7 +193,24 @@ function updateURL() {
     } else {
         url.searchParams.delete("tags");
         url.searchParams.delete("suite");
-        url.searchParams.set("suites", selectedSuites.join(","));
+        // Try finding common tags that would result in the current suite selection.
+        let commonTags = new Set(selectedSuites[0].tags);
+        for (const suite of Suites) {
+            if (suite.disabled) {
+                // commonTags = commonTags.difference(suite.tags);
+                suite.tags.forEach((tag) => commonTags.delete(tag));
```

Comment:
Just because this suite is disabled doesn't mean all the tags associated with this suite need to be removed / hidden, right? Those tags may have been associated with other suits that are enabled & appeared earlier in this loop.

Reply:
Currently we have to do it this way, as the `Suites.enable` method enables any suite that contains any of the provided flags (as opposed to only enabling it if all tags are present). (Up for changing that as well)

Reply:
Hm... I see. That's rather convoluted but I suppose we can address that in a follow up.